Tests/SwerverTests/ResponseSpec/ResponseFormatterSpec.swift

```diff
@@ -0,0 +1,35 @@
+import Quick
+import Nimble
+import Swerver
+import Foundation
+
+class ResponseFormatterSpec: QuickSpec {
+
+    override func spec() {
+        describe("A ResponseFormatter") {
+            var formatter: ResponseFormatter!
+            var responseBuilder: ResponseBuilder!
+            
+            beforeEach {
+                formatter = ResponseFormatter()
+                responseBuilder = ResponseBuilder()
+            }
+            
+            it("formats a Response object into a String object") {
+                let aResponse = responseBuilder
+                                    .setStatusCode(statusCode: StatusCode.ok.rawValue)
+                                    .setStatusPhrase(statusPhrase: StatusCode.ok.getStatusPhrase())
+                                    .setBody(body: "200 OK")
```

Comment:
Be careful about implementing more than you need. Nothing in the requirements talked about needing a body in the response. Adding features you do not need runs the risk of adding unneeded complexity.